docker/zookeeper.py

```diff
@@ -0,0 +1,148 @@
+import os
+import requests
+import boto3
+import logging as logger
+from pathlib import Path
+from configparser import ConfigParser
+
+DEFAULT_SERVER = '1=localhost:2888:3888;2181'
+
+# Get the environment variables
+def get_config_env():
+    ZOO_DATA_DIR = os.environ.get('ZOO_DATA_DIR')
+    ZOO_DATA_LOG_DIR = os.environ.get('ZOO_CONF_DIR')
+    ZOO_TICK_TIME = os.environ.get('ZOO_TICK_TIME', 2000)
+    ZOO_INIT_LIMIT = os.environ.get('ZOO_INIT_LIMIT', 5)
+    ZOO_SYNC_LIMIT = os.environ.get('ZOO_SYNC_LIMIT', 2)
+    ZOO_AUTOPURGE_SNAPRETAINCOUNT = os.environ.get('ZOO_AUTOPURGE_SNAPRETAINCOUNT', 3)
+    ZOO_AUTOPURGE_PURGEINTERVAL = os.environ.get('ZOO_AUTOPURGE_PURGEINTERVAL', 0)
+    ZOO_MAX_CLIENT_CNXNS = os.environ.get('ZOO_MAX_CLIENT_CNXNS', 60)
```

Comment:
I think we need to increase this number. In the context of prod, especially during deployment the number of nodes can go upto 60 ?

Reply:
These are configuration in the task definition.